addons/stock/models/stock_package.py

```diff
@@ -67,6 +121,18 @@ def _compute_valid_sscc(self):
             if package.name:
                 package.valid_sscc = check_barcode_encoding(package.name, 'sscc')
 
+    @api.onchange('package_type_id')
+    def _onchange_package_type(self):
+        if self.package_type_id and self.package_type_id.identification_method == 'auto':
+            self.name = '/'
+
+    def _search_contained_quant_ids(self, operator, value):
+        packages = self.search([('quant_ids', operator, value)])
+        if packages:
+            return [('id', 'parent_of', packages.ids)]
+        else:
+            return [('id', '=', False)]
+
```

Comment:
Would definitely do a kind of computed "package_line" from stock quand and non-done stock move line